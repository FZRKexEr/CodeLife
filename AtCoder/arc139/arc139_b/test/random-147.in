100
34 19 4 11 19 15
74 19 19 17 17 19
76 11 5 8 10 18
35 20 11 1 16 6
85 1 6 14 5 14
2 15 16 15 9 14
93 18 13 14 3 7
29 20 14 4 19 12
52 6 15 7 7 19
49 9 8 19 12 11
8 3 10 10 12 14
10 12 8 10 8 9
96 14 20 6 3 3
95 14 8 2 18 5
86 2 5 17 4 17
73 18 10 16 17 15
29 14 12 3 10 10
8 4 19 10 7 15
94 7 9 11 3 12
95 2 15 14 8 19
69 15 10 15 6 19
74 11 18 1 8 14
57 1 13 18 20 4
97 19 11 8 20 11
42 16 8 18 19 17
82 6 15 5 5 12
92 5 9 10 5 6
55 20 17 8 6 14
35 6 5 5 13 2
53 4 20 19 5 12
36 13 18 1 18 16
58 16 18 1 10 7
34 18 14 3 6 14
48 19 11 14 13 4
91 3 16 9 10 14
67 19 7 13 11 3
83 18 15 17 16 5
26 17 3 17 2 14
44 5 9 19 16 19
7 10 1 12 18 13
78 9 6 7 17 11
35 19 5 13 14 14
6 20 9 13 18 15
62 11 19 9 6 14
3 15 6 14 12 8
53 18 9 11 4 3
71 2 14 15 17 5
90 1 11 12 5 7
50 2 4 13 13 13
100 18 12 16 14 18
86 18 10 1 1 20
81 15 8 7 1 13
22 7 5 19 15 10
50 3 3 13 19 8
54 14 19 9 1 6
74 10 13 13 12 3
18 19 11 15 20 8
71 9 16 6 17 8
18 11 18 20 5 15
11 18 5 16 18 15
13 20 16 1 11 8
43 13 14 10 17 13
67 20 8 13 16 7
55 2 17 16 18 1
5 1 9 10 16 13
26 6 4 4 17 8
50 11 8 2 19 8
58 10 18 10 11 14
97 20 12 20 18 7
17 6 19 14 8 16
5 18 16 13 19 17
43 5 5 17 16 17
54 7 16 16 4 18
80 15 18 3 3 2
75 1 4 4 13 2
10 4 20 9 3 12
8 5 9 4 20 6
52 17 11 16 8 7
64 14 9 19 19 2
99 15 10 10 10 6
47 17 3 5 14 18
58 18 5 15 12 19
87 16 14 4 19 13
39 20 15 11 16 13
60 14 8 14 8 18
31 17 20 1 15 6
85 14 6 13 5 16
91 6 16 4 18 8
56 7 19 19 7 17
60 10 4 7 4 8
67 5 5 15 14 17
14 4 6 18 9 17
81 9 16 13 4 5
8 15 12 5 5 15
54 4 5 13 8 18
7 14 16 9 11 7
89 13 10 14 12 7
14 2 3 17 10 16
100 10 1 12 1 20
12 10 16 4 13 18
