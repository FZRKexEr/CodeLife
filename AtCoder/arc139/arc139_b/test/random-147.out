97
340
96
20
201
30
30
28
68
67
44
29
18
69
172
84
23
14
59
144
116
55
106
105
91
70
68
42
14
58
32
25
26
72
118
47
81
44
131
84
143
69
78
58
42
99
40
66
169
156
18
47
45
154
31
62
113
74
15
46
13
49
92
39
50
56
52
58
76
86
65
179
64
20
48
24
32
40
33
98
93
96
88
73
88
17
43
78
56
24
212
35
36
40
124
63
71
70
10
21
